<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#191A19;
background-color:rgb(48, 48, 48);
QPushButton{
color:blue;
}
QTabBar::tab {
    margin-left:40px;
    margin-right:40px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>51</width>
      <height>62</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 22px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>123214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabBar::tab {
    margin-left:70px;
	margin-right:10px;
	font: 18pt &quot;Segoe Print&quot;;
  font-weight: Bold;

color:black;
background-color: rgb(227, 227, 227);

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    border-bottom-left-radius: 10px;
    min-width: 10ex;
    padding: 1px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
color:rgb(209, 144, 40);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
background-color: rgb(42, 116, 53);
}

QTabBar::tab:!selected {
    margin-top: 4px; /* make non-selected tabs look smaller */
}
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>33</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="weapons">
     <attribute name="title">
      <string>weapons</string>
     </attribute>
     <widget class="QGraphicsView" name="weapon_view">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(220, 220, 220);
background-color: rgb(42, 116, 53);</string>
      </property>
     </widget>
     <widget class="QLabel" name="bomb_price_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 34px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
     <widget class="QLabel" name="bomb_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
      </property>
      <property name="text">
       <string>bomb image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Buy_bomb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
 QPushButton#Buy_bomb {
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #white;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
background-color: #1e1d23;
 }

 
QPushButton#Buy_bomb:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton#Buy_bomb:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 2px;
    margin-top: 4px;     
	background-color: #1e1d23;
	background-color: rgb(126, 126, 126);
}
QPushButton#Buy_bomb:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;

}</string>
      </property>
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="weapon_view2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(220, 220, 220);
background-color: rgb(42, 116, 53);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Buy_powerfulbullet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
 QPushButton#Buy_powerfulbullet {
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #white;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
background-color: #1e1d23;
 }

 
QPushButton#Buy_powerfulbullet:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton#Buy_powerfulbullet:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 2px;
    margin-top: 4px;     
	background-color: #1e1d23;
	background-color: rgb(126, 126, 126);
}
QPushButton#Buy_powerfulbullet:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;

}</string>
      </property>
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
     <widget class="QLabel" name="powerfulbullet_price_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 34px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
      </property>
      <property name="text">
       <string>400</string>
      </property>
     </widget>
     <widget class="QLabel" name="pwerfulbullet_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
      </property>
      <property name="text">
       <string>powerfull bullet image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shields">
     <attribute name="title">
      <string>shields</string>
     </attribute>
     <widget class="QGraphicsView" name="weapon_view2_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(220, 220, 220);
background-color: rgb(42, 116, 53);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Buy_health">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
 QPushButton#Buy_health {
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #white;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
background-color: #1e1d23;
 }

 
QPushButton#Buy_health:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton#Buy_health:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 2px;
    margin-top: 4px;     
	background-color: #1e1d23;
	background-color: rgb(126, 126, 126);
}
QPushButton#Buy_health:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;

}</string>
      </property>
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
     <widget class="QLabel" name="health_image">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
      </property>
      <property name="text">
       <string>health_image</string>
      </property>
     </widget>
     <widget class="QLabel" name="health_price">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 34px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="weapon_view2_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(220, 220, 220);
background-color: rgb(42, 116, 53);</string>
      </property>
     </widget>
     <widget class="QLabel" name="health_price_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 34px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
     <widget class="QLabel" name="drink_image">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
      </property>
      <property name="text">
       <string>drink-image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Buy_health_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
 QPushButton#Buy_health_2 {
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #white;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
background-color: #1e1d23;
 }

 
QPushButton#Buy_health_2:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton#Buy_health_2:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 2px;
    margin-top: 4px;     
	background-color: #1e1d23;
	background-color: rgb(126, 126, 126);
}
QPushButton#Buy_health_2:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;

}</string>
      </property>
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="coins">
     <attribute name="title">
      <string>coins</string>
     </attribute>
     <widget class="QGraphicsView" name="weapon_view2_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(220, 220, 220);
background-color: rgb(42, 116, 53);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Buy_bomb_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
 QPushButton#Buy_bomb_3 {
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #white;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
background-color: #1e1d23;
 }

 
QPushButton#Buy_bomb_3:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton#Buy_bomb_3:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 2px;
    margin-top: 4px;     
	background-color: #1e1d23;
	background-color: rgb(126, 126, 126);
}
QPushButton#Buy_bomb_3:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;

}</string>
      </property>
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
     <widget class="QLabel" name="bomb_price_label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
border:none;
 font-family: Arial;
  font-size: 34px;
  font-weight: Bold;
color:rgb(209, 144, 40);</string>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
     <widget class="QLabel" name="coins_image1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
      </property>
      <property name="text">
       <string>coins_image 1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="coins_image">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-image:url(qrc:/assets/images/button.png);
background:none;
border:none;</string>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <zorder>ExitButton</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label</zorder>
   <zorder>coins_image</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
